### DISCUSSION TITLE
Where do Sent Feedback go?

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#708] [Discussion]
I want to know what people have been saying and what bugs they've caught... When I go to https://scratchaddons-feedback.glitch.me/ all it says is "hi! lol"

**Comment by mxmou** [comment]
I think it's posted in the Discord channel, but I'm not sure because I don't use Discord.

**Reply by TheColaber** [reply]
Me neither...

**Reply by TheColaber** [reply]
Where tho?

**Reply by mxmou** [reply]
> Where tho?

If you mean where the Discord channel is: the link is in the readme.

**Reply by TheColaber** [reply]
No, I mean where is it located in the discord? I can only see #welcome...

**Reply by mxmou** [reply]
> No, I mean where is it located in the discord? I can only see #welcome...

I don't know.

**Reply by GrahamSH-LLK** [reply]
@TheColaber It's a hidden channel. Who are you on discord? I'll verify you.
Edit: Duh, you're TheColaber. You are verified. Only some users can see feedback, though.

**Reply by Explosion-Scratch** [reply]
Can I have access? I've been with this project literally forever.

**Comment by WorldLanguages** [comment]
It's a hidden channel in the Discord server only some of us can access.

**Reply by WorldLanguages** [reply]
Note: anyone can access the Discord server! I meant to say, not everyone can read the sent feedback.

**Reply by TheColaber** [reply]
Oh.
